### PR TITLE
tests: add insert bar tests

### PR DESCRIPTION
**Pull request by naterush**
# Description

Adds more tests. Took < 30 minutes. Very nice. 

# Testing

See tests.

# Documentation

No.

### CODE REVIEW

File: tests/playwright.config.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ exports = require('@jupyterlab/galata/lib/playwright-config');
 
 module.exports = {
     ...exports,
+    timeout: 15_000,
```
Comment (naterush) [inline tests/playwright.config.js:8]: Oh, this is probably breaking JupyterLab tests

**Comment by marthacryan** [comment]
Should we start splitting these into smaller files? This file is getting a little unweildy

File: tests/streamlit_ui_tests/streamlit.spec.ts [hunk 2]
```diff
@@ -320,6 +335,161 @@ test.describe('Home Tab Buttons', () => {
...
+  test('Test Configure Code', async ({ page }) => {
+    const mito = await getMitoFrameWithTestCSV(page);
+    await clickTab(page, mito, 'Code');
+
+    await mito.getByRole('button', { name: 'Configure Code' }).click();
+    await checkOpenTaskpane(mito, 'Generated Code Options');
+
+    await mito.locator('.toggle').first().click();
+    await mito.getByRole('textbox').fill('new name');
+    // Wanna check some output
```
Comment (marthacryan) [inline tests/streamlit_ui_tests/streamlit.spec.ts:472]: Doesn't look like you're checking the output?

**Comment by marthacryan** [comment]
These look great! I'm loving these utilities - the tests are so easy to read! Other than that one unfinished test LGTM.

**Comment by naterush** [comment]
> Should we start splitting these into smaller files? This file is getting a little unwieldy

I think soon! We can do that in this weeks robustness refactors.

File: tests/streamlit_ui_tests/streamlit.spec.ts (hunk 2 above)
Comment (naterush) [reply tests/streamlit_ui_tests/streamlit.spec.ts:472]: Yeah, per comment I think we can ignore the code tests.

**Pull request merged**